README.md

```diff
@@ -479,6 +479,8 @@ Here are the settings that you can configure by default:
 - `headers` - the headers of your requests. Default value is `{ 'content-type': 'application/json' }`.
 - `documentValidation` - whether the EPCISDocument has to be validated or not before sending it via the Capture
   interface. The default value is `true`.
+- `checkExtensions` - if it is set to true the extension of the EPCISDocument will be checked against the EPCIS Document context.
```

Comment:
At the end of the first line, I would add something like:
`(e.g verifying if all the prefixes are defined in the context of the EPCISDocument)`
To explain what it means to check an extension